app/controllers/local_transaction_controller.rb

```diff
@@ -96,21 +94,8 @@ def lgil
     content_item["details"]["lgil_code"] || content_item["details"]["lgil_override"]
   end
 
-  def local_authority
-    if interaction_details["local_authority"]
-      local_authority = LocalAuthorityPresenter.new(interaction_details["local_authority"])
-    end
-
-    unless interaction_details["local_interaction"]
-      @location_error = error_for_missing_interaction(local_authority)
-    end
-
-    local_authority
-  end
-
   def interaction_details
     council = params[:local_authority_slug]
```

Comment:
will it always have a params[:local_authority_slug]? I guess the validation errors in the publishing app mean that it would?

Reply:
This interaction details method is only called by the results action which needs a `local_authority_slug` param